bot/cogs/define_cog.py

```diff
@@ -0,0 +1,150 @@
+#This contribution was made by: Rajat Sethi
+#Date: 12/15/2020
+
+import logging
+
+import discord
+import discord.ext.commands as commands
+from bot.messaging.events import Events
+from bot.bot_secrets import BotSecrets
+
+import json
+import requests
+
+
+log = logging.getLogger(__name__)
+
+class defineCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.api_key = BotSecrets.get_instance().merriam_key
+        self.wordPages = []
+    
+    def getPageData(self, response, word):
+        jsonData = response.json()
+        pages = []
+
+        #If the word is found, the JSON will return a dictionary of information. 
+        if (isinstance(jsonData[0], dict)):
+
+            #For words with several definitions, it will return several dictionaries. 
+            for i in range(len(jsonData)):
+
+                #Current dictionary of information.
+                wordData = jsonData[i]
+
+                #Stems of the given word (Past Tense, Future Tense, Perfect Tense, etc.)
+                wordStems = []
+                try:
+                    wordStems = wordData['meta']['stems']
+                except:
```

Comment:
all these try excepts should be switched to a .get which allows you to define a default value to be returned if a key doesnt exist